### PR TITLE
#169654249 bug fix tagging issue on creating articles

### PR DESCRIPTION
**Pull request by nshutijonathan**
#### What does this PR do?
-It adds tagging functionality
#### Description of Task to be completed?
- Added field of tags in the articles table 
#### How should this be manually tested?
- clone the repo the 
#### Any background context you want to provide?
- Download or clone the tutorial project code from [here](https://github.com/andela/codepirates-ah-backend), in your terminal then git checkout ` bg-fix-user-tag-on-creating-article-169654249 ` , install all required NPM packages by running NPM install from the command line in the project root folder (where the package.json is located).

- Start the API by running NPM start from the command line in the project root folder, you should see the message Server listening on port 8000. You can test the API directly using an application such as Postman or you can test it with one of the single pages in the browser.
- to test authentication and authorization, add this to the header
Key: :x-access-token,value:JWT TOKEN
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/169654249
#### Screenshots (if appropriate)

image

### CODE REVIEW

File: src/migrations/20190813125249-create-article.js [hunk 1]
```diff
@@ -10,19 +10,23 @@ module.exports = {
...
         type: Sequelize.STRING,
-        allowNull: false,
+        allowNull: false
       },
       description: {
         type: Sequelize.TEXT,
-        allowNull: false,
+        allowNull: false
+      },
+      taglist: {
```
Comment (salviosage) [inline src/migrations/20190813125249-create-article.js:23]: I do think tag list has to be an array, not a string

File: src/controllers/articles.controller.js [hunk 2]
```diff
@@ -62,7 +63,8 @@ class Articles {
           favoritedcount: createdArticle.favoritedcount,
           images: createdArticle.images,
           createdAt: createdArticle.createdAt,
-          updatedAt: createdArticle.updatedAt
+          updatedAt: createdArticle.updatedAt,
+          taglist: createdArticle.taglist
```
Comment (salviosage) [inline src/controllers/articles.controller.js:69]: I think there is a tag list table where they have to be saved, I suggest making use of it.

**Review by salviosage — CHANGES_REQUESTED** [review]
hi @nshutijonathan , nice work. I have left some comment

File: src/controllers/articles.controller.js (hunk 2 above)
Comment (nshutijonathan) [reply src/controllers/articles.controller.js:69]: I was requested to add taglist as I create articles and then, save articles along with their tags. Thank you for the feedback

File: src/migrations/20190813125249-create-article.js (hunk 1 above)
Comment (nshutijonathan) [reply src/migrations/20190813125249-create-article.js:23]: It is now an array

File: src/controllers/articles.controller.js (hunk 2 above)
Comment (nshutijonathan) [reply src/controllers/articles.controller.js:69]: in addition I can't save data from front-end indifferent tables

**Pull request merged**